docs/validators/protocol_nodes/pools/archway_state-sync/run_ksync.md

```diff
@@ -50,9 +50,9 @@ You can also install from source by pulling the ksync repository and switching t
 as follows:
 
 ```bash
-git clone git@github.com:KYVENetwork/ksync.git
+git clone https://github.com/KYVENetwork/ksync.git
 cd ksync
-git checkout tags/v1.0.0-beta.0 -b v1.0.0-beta.0
+git checkout tags/v1.0.0-beta.0 -b v1.0.0-beta.2
```

Comment:
Please remove or update these change due to a merge conflict with the current main branch. Thanks!